_layouts/default.html

```diff
@@ -4,15 +4,18 @@
 
   <body>
 
-    <div>
-      <a href="{{ site.baseurl }}/">
-        <header>
-          <img src="{{ site.baseurl }}/img/logo.svg" alt="{{ site.title }}" />
-        </header>
-      </a>
-
-      {{ content }}
-
+    <div id="wrapper">
+      <div id="content">
+        <div>
```

Comment:
@MattiSG `<main>` is for [content](https://developer.mozilla.org/en/docs/Web/HTML/Element/main) without header, footers, logos. So I changed the class name and I removed the useless `div`